src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -767,9 +767,9 @@ internal override unsafe void GetEHInfo(int excNumber, void* exc)
 
         internal override void ResolveToken(int token, out IntPtr typeHandle, out IntPtr methodHandle, out IntPtr fieldHandle)
         {
-            typeHandle = new IntPtr();
-            methodHandle = new IntPtr();
-            fieldHandle = new IntPtr();
+            typeHandle = default;
+            methodHandle = default;
+            fieldHandle = default;
```

Comment:
💭 I would normally use `IntPtr.Zero` for these cases

Reply:
StyleCop's auto-fixer doesn't do that.  This was an almost entirely automated change.

Reply:
👍 Thanks I filed DotNetAnalyzers/StyleCopAnalyzers#3065 as a result.